MultiLangProgGuide/MultiLangProgGuide.rst

```diff
@@ -1553,18 +1555,14 @@ ydb_fork_n_core()
 
 	void ydb_fork_n_core(void)
 
-The core of a process can help debug application code, for example to
-troubleshoot an out-of-design condition. However, generating a core
-terminates the process, and furthermore, if the process is last
-process with a database file open, that file can be left in an
-“unclean” state, even if a state that is easily recovered; and
-cleaning up open database files before generating the core perturbs
-the process state to be debugged.
-
-When a process executes :code:`ydb_fork_n_core()`, the process forks. The
-child process does essential cleanup (such as erasing encryption
-passphrases) and sends itself a signal to generate a core and
-terminate, while the parent process continues execution.
+A core is a snapshot of a process, to help debug application code, for
+example to troubleshoot an out-of-design condition.  When a process
+executes :code:`ydb_fork_n_core()`, it forks. The child process does
```

Comment:
The child process does NO cleanup. If encryption passwords are in memory, they become part of the core also. The core file is supposed to be a "virginal" copy of memory at the time of the call so we make that true as much as possible with as little processing as possible since this may be happening as part of something out-of-design that occurred.